src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/EventHandlerModifier/EventHandlerModifierTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.PrivateEventHandlers;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Acuminator.Utilities;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.EventHandlerModifier
+{
+	public class EventHandlerModifierTests : DiagnosticVerifier
+	{
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => new PXGraphAnalyzer(
+				CodeAnalysisSettings.Default
+									.WithRecursiveAnalysisEnabled()
+									.WithStaticAnalysisEnabled()
+									.WithSuppressionMechanismDisabled(),
+				new EventHandlerModifierAnalyzer());
+
+		[Theory]
+		[EmbeddedFileData("PrivateModifier.cs")]
+		public void PrivateModifierNotAllowed(string source)
```

Comment:
The test method should return Task, await VerifyCSharpDiagnosticAsync and be async methods

Reply:
Done.